### DISCUSSION TITLE
Lsposed module development requirements

### DISCUSSION THREAD

**Discussion by r1shhh1** [LSPosed/LSPosed#2479] [General]
I am trying to build a android id spoofer that fakes the ids of the android device .
My module is being recognised by lsposed but I'm unable to enable it through lsposed .
Would really appreciate some help

**Comment by elesbb** [comment]
Do you have your `xposed_init` file in the assets folder for your apk? Then in the `xposed_init` file you enter the full package name and class of the xposed class file: `com.example.xposed_test.XposedMain` the "XposedMain" has the hooks in it.
